doc/readme/EXAMPLES.md

```diff
@@ -94,6 +94,16 @@ You can render it with
 which is also part of `make doc`.
 The target might not be available if the requirements in `doc/cookbook/requirements.txt` are not satisfied (in particular Sphinx), or if the meta examples are disabled.
 
+If the following error is still visible
+
+    make: *** No rule to make target 'cookbook'. Stop.
+
+Find out if value of `SPHINX_EXECUTABLE` is set by searching for it in CMakeCache.txt which is present in the build directory. If it's not set then not all requirements in `doc/cookbook/requirements.txt` have been satisfied. 
+
+If it is set then try to compile by giving a explicit path to `sphinx-build` file. The file can be located by the command `locate sphinx-build` and then use the following cmake statement to build shogun again and then try to render the cookbooks.
+
+    cmake -DSPHINX_EXECUTABLE="<path/to/sphinx-build>" -DBUILD_META_EXAMPLES=On [other cmake options] ..
```

Comment:
`-DBUILD_META_EXAMPLES=ON`, no? Also shouldn't you be using \``` ``` for code blocks?

Reply:
For making the cookbook, it is necessary for BUILD_META_EXAMPLES to be On. So I felt it should be added.
I was just following the convention followed in the rest of the readme with respect to code blocks.

Reply:
Should I remove the BUILD_META_EXAMPLES?

Reply:
I guess what gf712 means is that you should use `-DBUILD_META_EXAMPLES=ON` instead `-DBUILD_META_EXAMPLES=On`, `On` should be `ON`.

Reply:
Oh, My bad. I was using "On" the entire time lacally, didn't notice. I have changed it. Thanks @LiuYuHui